src/util/util.c

```diff
@@ -166,7 +186,7 @@ void chomp(char *str) {
         for ( i = 1; i < len; i++ ) {
 	    str[i-1] = str[i];
 	}
-	str[len] = '\0';
+	str[len-1] = '\0';
```

Comment:
This should probably be merged separately and ASAP, I think it's possible to attempt to write that null byte out of allocated memory.